lib/Db/MessageMapper.php

```diff
@@ -508,6 +508,9 @@ public function updatePreviewDataBulk(Message ...$messages): array {
 				$previewText = null;
 				if ($message->getPreviewText() !== null) {
 					$previewText = mb_strcut(mb_convert_encoding($message->getPreviewText(), 'UTF-8', 'UTF-8'), 0, 255);
+
+					//converting the spaces is necessary for ltrim to work
```

Comment:
```suggestion
						$convertedText = mb_convert_encoding($message->getPreviewText(), 'UTF-8', 'UTF-8');
					//converting the spaces is necessary for ltrim to work
					$previewText = mb_strcut(ltrim(preg_replace('/\s/u', ' ', $convertedText)), 0, 255);

					// Make sure modifications are visible when these objects are used right away
					$message->setPreviewText($previewText);
```
I'm still not sure why we're converting from UTF-8 to UTF-8,
but I think this way we can keep the 255 characters limit in case of an edge-case where a message starts with a huge amount of spaces